### DISCUSSION TITLE
Enable "Make Link" option by default for Nemo file manager

### DISCUSSION THREAD

**Discussion by SpeeQz1** [linuxmint/discussions#649] [Ideas]
# Suggestion
I've seen a couple of windows users struggling with moving over to Mint due to them not having the "Make Link" option enabled by default and not knowing how to make the shortcuts.

I propose that the option for making a link is enabled by Default as it would makes it in general easier for any user to not have to manually enable it and it is quite crucial to have in the first place.

![image](https://github.com/user-attachments/assets/726a3634-ee27-48d7-97fc-06428cb30559)

**Comment by anaximeno** [comment]
For those who want to create a direct shortcut to the desktop, there's already a downloadable action named 'Create Desktop Shortcut Action' in the Actions settings module or from the cinnamon spices website https://cinnamon-spices.linuxmint.com/actions/view/11

**Comment by Jopp-gh** [comment]
Welcome on Linux 🙂

> I've seen a couple of windows users struggling with moving over to Mint due to them not having the "Make Link" option enabled by default

Ehm, this sounds a bit dramatic however it's true. Roughly, Windows shares the same hardware with Linux from a user perspective. 

Many features in Mint help switchers to find a start - so newcomers obviously look for similar or same features they know from windows and some new users may even think that Mint is just some recycled "older code" originating from  windows.

**Comment by haggen88** [comment]
https://github.com/linuxmint/nemo/issues/2883
